### DISCUSSION TITLE
mac桌面版阴阳师点开没反应

### DISCUSSION THREAD

**Discussion by GamhimLew** [lisai9093/YYS#9] [Q&A]
大大
我浅浅研究下（python小白）
我们Mac用户是不是可以通过自己截图然后替换png文件夹上的对应截图，实现自适配？

**Comment by lisai9093** [comment]
可以的。如果是用crossover的话应该可以不用修改直接用。但是m1/m2用商店版本的话因为分辨率不同所以需要重新截图。

**Comment by GamhimLew** [comment]
已收到您的邮件祝生活愉快！Your email has been receivedWish you a happy life!メールを受信しました楽しい生活をお過ごしください！Hin.
